pkg/gds/admin.go

```diff
@@ -778,7 +778,10 @@ func (s *Admin) acceptRegistration(vasp *pb.VASP) (msg string, err error) {
 	for careqs.Next() {
 		req := careqs.CertReq()
 		if req != nil && req.Vasp == vasp.Id && req.Status == models.CertificateRequestState_INITIALIZED {
-			req.Status = models.CertificateRequestState_READY_TO_SUBMIT
+			// TODO: Replace "email" in the source parameter with user email address.
```

Comment:
@pdeziel I'd like to merge this in to help me with SC-909; would you mind creating a story to make this small update?